template/bash_profile.template.bash

```diff
@@ -37,7 +37,11 @@ export TODO="t"
 
 # Set vcprompt executable path for scm advance info in prompt (demula theme)
 # https://github.com/xvzf/vcprompt
-#export VCPROMPT_EXECUTABLE=~/.vcprompt/bin/vcprompt
+# export VCPROMPT_EXECUTABLE=~/.vcprompt/bin/vcprompt
 
 # Load Bash It
 source $BASH_IT/bash_it.sh
+
+# Retain current working directory when opening a new tab
+# https://github.com/revans/bash-it/issues/240
+# export PROMPT_COMMAND="$PROMPT_COMMAND;update_terminal_cwd;"
```

Comment:
The question is whether the function will also work in iTerm2.

Here are a couple of things we could do:

* If the `update_terminal_cwd` function is defined, add it to the prompt command, or
* If the function is not defined, define it (this could be done in a plugin), and then add it to the prompt command

The first one should be easy, for the second one I'd like to ask for opinions...